packages/pwa-kit-react-sdk/src/ssr/browser/main.jsx

```diff
@@ -132,8 +132,17 @@ export const start = async () => {
         locals
     })
 
+    const _routes = await getAllRoutes(locals)
+
+    applicationExtensions.forEach((extension) => {
+        const componentMap = extension.getComponentMap(_routes)
+        console.log('JINSU componentMap', componentMap)
+        extension.deserialize(componentMap)
+    })
+
     const routes = await getAllRoutes(locals)
```

Comment:
we need to call `getAllRoutes` twice. First time is to get the routes from the other extensions so we can pass it to `getComponentMap` and deserialize. The 2nd time is to reload the routes that have been deserialized from cache.

I also thought of loading only the synchrounous routes before deserializing and then loading the async routes after deserializing but i don't think that is viable since it will mess up the order of the routes.

Reply:
prioritization and put it the routes at the end
for now we can hard code it to put SEO at the end
or assume it ats the end

getComponentMap only get routes from extension before it
manually process seo one last

deserialize first

routes = [] => deserialize empty ext and add routes into it